src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -226,13 +227,36 @@ export default class ReimbursementRequestService {
     return deletedRequest;
   }
 
+  /**
+   * Returns all reimbursement requests that do not have an advisor approved reimbursement status.
+   * @param requester the user requesting the reimbursement requests
+   * @returns reimbursement requests with no advisor approved reimbursement status
+   */
+  static async getPendingAdvisorList(requester: User): Promise<ReimbursementRequest[]> {
+    await validateUserIsHeadOfFinanceTeam(requester);
+
+    const requestsPendingAdvisors = await prisma.reimbursement_Request.findMany({
+      where: {
+        saboId: { not: null },
+        reimbursementsStatuses: {
+          none: {
+            type: Reimbursement_Status_Type.ADVISOR_APPROVED
```

Comment:
damn, we actually want this to be any request with the status SABO_SUBMITTED (aka Pending_advisor), see the pending advisor list section of https://github.com/Northeastern-Electric-Racing/FinishLine/issues/493. I'm just gonna fix it since I left this PR for so long 💀